tests/framework/components/sprite/test_sprite_component.js

```diff
@@ -67,7 +67,22 @@ test('Add new Component', function () {
     ok(e.sprite);
 });
 
-test('Add / Remove Component', function () {
+// TODO This and other tests in this file are skipped because of the following error:
+//
+//      Uncaught TypeError: Cannot read property 'length' of undefined
+//        at SpriteAnimationClip._setFrame (sprite-animation-clip.js:183)
+//        atSpriteAnimationClip.set (sprite-animation-clip.js:381)
+//        atSpriteAnimationClip.set (sprite-animation-clip.js:368)
+//        atSpriteAnimationClip._onSpriteAssetLoad (sprite-animation-clip.js:84)
+//        atSpriteAnimationClip._bindSpriteAsset (sprite-animation-clip.js:56)
+//        atSpriteAnimationClip.set (sprite-animation-clip.js:298)
+//        atSpriteComponent.set (component.js:586)
+//        atSpriteComponentSystem.initializeComponentData (system.js:149)
+//        atSpriteComponentSystem.addComponent (system.js:84)
+//        atEntity.addComponent (entity.js:116)
+//
+// Once this has been addressed they can be re-enabled.
```

Comment:
These tests were passing before, but only because one of the files was actually failing to load and so the significant parts of the tests weren't running:

<img width="939" alt="screen shot 2018-07-29 at 17 10 05" src="https://user-images.githubusercontent.com/37333094/43368447-1fae3ea8-9355-11e8-8f8f-3c8793c4fdc6.png">

After I fixed the test asset path I started getting the error:

<img width="939" alt="screen shot 2018-07-29 at 17 10 23" src="https://user-images.githubusercontent.com/37333094/43368452-31b78ee2-9355-11e8-911d-e292681128a9.png">